src/validation.js

```diff
@@ -0,0 +1,36 @@
+import ERROR_MESSAGE from "./constants/errorMessage.js";
+
+export const validateCarNameList = (carNameListProp) => {
+  const WRONG_PATTERN = /[^ㄱ-ㅎ가-힣A-Za-z,]/g;
+
+  const wrongPattern = carNameListProp.match(WRONG_PATTERN);
```

Comment:
```suggestion
  const wrongCarName = carNameListProp.match(WRONG_PATTERN);
```
`WRONG_PATTERN`에 포함되는 자동차를 나타내는 변수이니깐 `wrongPattern`이라는 네이밍보다는 `wrongCarName`같은 건 어떨까요??